### PR TITLE
remove ios 7 logic as sdk minimum support is iOS 8 now

### PR DESCRIPTION
**Pull request by chliangGoogle**

### CODE REVIEW

File: Firebase/InstanceID/FIRInstanceIDURLQueryItem.h [hunk 1]
```diff
@@ -18,7 +18,6 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-// Stand-in for NSURLQueryItem, which is only available on iOS 8.0 and up.
 @interface FIRInstanceIDURLQueryItem : NSObject
```
Comment (paulb777) [inline Firebase/InstanceID/FIRInstanceIDURLQueryItem.h:21]: Could the clients use NSURLQueryItem instead and delete the whole class?

Comment (chliangGoogle) [reply Firebase/InstanceID/FIRInstanceIDURLQueryItem.h]: Good idea! Didn't realize that.

File: FirebaseMessaging/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# unreleased
+- [changed] Remove logic that is executed for iOS 7 and below. (5835)
```
Comment (paulb777) [inline FirebaseMessaging/CHANGELOG.md:2]: Missing '#'

Comment (chliangGoogle) [reply FirebaseMessaging/CHANGELOG.md]: Done.

**Pull request merged**